README.md

```diff
@@ -456,3 +455,120 @@ ansible -i inventory.yaml overcloud -a "sudo dnf -y install python3 lvm2"
 cd
 ./tripleo-deploy.sh
 ```
+Remove an overcloud VM or baremetal host
+----------------------------------------
+
+Removing an overcloud VM or baremetal host requires the following steps:
+
+## Disable the compute service
+
+In case a compute node gets removed, disable the Compute service on the outgoing node on the overcloud to prevent the node from scheduling new instances
+
+```bash
+openstack compute service list
+openstack compute service set <hostname> nova-compute --disable
+```
+
+## Annotate the VM/BMH resource for deletion
+
+Annotation of a VM resource
+
+```bash
+oc annotate -n openstack vm/controller-1 osp-director.openstack.org/delete-host=true --overwrite
+```
+
+Annotation of a BMH resource
+
+```bash
+oc annotate -n openshift-machine-api bmh/openshift-worker-3 osp-director.openstack.org/delete-host=true --overwrite
+```
+
+The annotation status is being reflected in the OSBaremetalset/OSVMset using the `annotatedForDeletion` parameter:
+
+```bash
+oc get osbms computehci -o json | jq .status
+{
+  "baremetalHosts": {
+    "computehci-0": {
+      "annotatedForDeletion": true,
```

Comment:
should we reflect this in the new status summary, either annotated for deletion count, or list of names?

Reply:
Hmm.  If the list doesn't make for too much clutter, we could do that.  If it is, then perhaps a count would work.

Reply:
yes that's the only concern that the list could get to long